docs/section-7/drivetrain-characterization.md

```diff
@@ -0,0 +1,83 @@
+
+![DC Motor](https://upload.wikimedia.org/wikipedia/commons/thumb/e/e6/Electric_motor_cycle_1.png/200px-Electric_motor_cycle_1.png)
+
+![Spinning DC Motor](https://upload.wikimedia.org/wikipedia/commons/thumb/8/89/Electric_motor.gif/200px-Electric_motor.gif)
+
+But what prevents the motor from accelerating indefinitely? Well, just as current running through the motor causes the armature to spin, spinning the armature generates current. So, while the motor is spinning, a force, known as the counter electromotive force (or back-EMF), restricts the flow of current. Eventually, back-EMF will cancel out the torque generated by the windings and the rotational speed of the motor will reach a constant value.
+
+![Speed Graph](speed_curve.PNG)
+
+Kirchhoff's voltage law states that the sum of the voltages in any closed loop in a circuit is zero. There are three voltages: a voltage we wish to apply \(V_{app}\), a voltage drop across the windings \(V_{windings}\), and a voltage drop due to back-EMF \(V_{emf}\). If we apply Kirchhoff's voltage law, we get:
+
+\[V_{app} - V_{emf} - V_{windings} = 0\]
+
+Which, by solving for \(V_{app}\), becomes:
+
+\[V_{app} = V_{emf} + V_{windings}\]
+
+!!! note
+    It is known that \(V_{windings} = IR\) where \(I\) is the current and \(R\) is the resistance and \(I \propto \tau\) where \(\tau\) is the motor torque. Additionally \(V_{emf} \propto \omega\) where \(\omega\) is the rotational speed of the motor. Since \(\tau \propto \frac{d\omega}{dt}\),
+    we can create an equation for \(\omega(t)\):
+        \[V_{app} = a\frac{d\omega}{dt} + b\omega(t)\]
+    where \(a\) and \(b\) are constants. For those who are inclined to do some calculus, you can find an equation relating rotational speed with time. The behavior of this function should match the behavior described above.
+
+The voltage due to back-EMF varies directly with the speed of the wheels and therefore varies directly with the speed of the drivetrain, or \(V_{emf} = k_v * \text{velocity}\). The voltage across the windings causes torque, which is proportional to the acceleration of the wheels and therefore the acceleration of the drivetrain, meaning \(V_{windings} = k_a * \text{acceleration}\). Finally, we need to account for any resistance we encounter in trying to move, such as friction. It so happens that this resistance is constant no matter the deisred velocity and acceleration. All that needs to be done is add a third constant - let's call it \(k_{volt}\) (other sources may call it \(k_S\)). Our new formula becomes:
```

Comment:
"this resistance is constant" -> "this resistance is approximately constant"

"deisred" -> "desired"